test/extended/util/test.go

```diff
@@ -210,32 +208,23 @@ func createTestingNS(baseName string, c kclientset.Interface, labels map[string]
 		baseName = "e2e-" + baseName
 	}
 
+	// we skip SCC if this is a kube e2e test
+	isKubeE2ENamespace := strings.HasPrefix(baseName, "e2e-k8s-") || (isKubernetesE2ETest() && !skipTestNamespaceCustomization())
+	if isKubeE2ENamespace{
+		labels["security.openshift.io/disable-securitycontextconstraints"] = "true"
```

Comment:
is this safe? or should it check that `labels` is not nil to avoid panicing?

Reply:
I see some places that are setting labels to nil IIUIC
https://github.com/openshift/origin/blob/aa7e2387a3c8ac48293307340fbe5c40fc186577/vendor/k8s.io/kubernetes/test/e2e/apimachinery/namespace.go#L89

Reply:
agreed :+1: